pkg/container/vector/vector.go

```diff
@@ -342,6 +342,61 @@ func GetArrayAt[T types.RealNumbers](v *Vector, i int) []T {
 	return types.GetArray[T](&bs[i], v.area)
 }
 
+func GetAny(vec *Vector, i int) any {
```

Comment:
It is not recommended to add this GetAny interface, the use of `any` will have additional on-heap memory allocation, causing stress on the GC. Similarly the use of AppendAny is not recommended.

Reply:
comment added for GetAny and AppendAny to warn that memory will escape to heap for any type that result in slow GC.